data/systems/partial/02_local_stars.json

```diff
@@ -1,3 +1,409 @@
+    {"name": "Gliese 773.6", "stars": ["STAR_K"], "sector": [-3, -2, -4], "pos": [2.736, 3.408, 2.496]},
+    {"name": "NN 3855", "stars": ["STAR_M"], "sector": [-2, 1, 3], "pos": [6.264, 4.6, 3.264]},
+    {"name": "GJ 1105", "stars": ["STAR_M"], "sector": [2, 1, 2], "pos": [1.392, 1.856, 1.568]},
+    {"name": "GJ 1103", "stars": ["STAR_M", "STAR_M"], "sector": [3, 1, 0], "pos": [1.312, 5.448, 0.0]},
+    {"name": "NN 3454", "stars": ["STAR_M"], "sector": [2, 1, 0], "pos": [2.24, 0.16, 2.48]},
+    {"name": "NN 3325", "stars": ["STAR_K"], "sector": [3, -1, -2], "pos": [3.984, 0.936, 6.968]},
+    {"name": "NN 3323", "stars": ["STAR_M"], "sector": [2, -1, -1], "pos": [3.232, 3.024, 5.584]},
+    {"name": "GJ 1243", "stars": ["STAR_M"], "sector": [-3, -2, 3], "pos": [0.312, 3.512, 4.192]},
+    {"name": "NN 3459", "stars": ["STAR_M"], "sector": [3, 1, -2], "pos": [5.984, 5.768, 3.184]},
+    {"name": "NN 3592", "stars": ["WHITE_DWARF"], "sector": [1, 2, -4], "pos": [0.92, 2.096, 1.896]},
+    {"name": "NN 4063", "stars": ["STAR_M"], "sector": [-3, -1, 2], "pos": [4.12, 4.976, 0.952]},
+    {"name": "NN 4062", "stars": ["STAR_M"], "sector": [-4, -1, 3], "pos": [0.992, 3.64, 2.912]},
+    {"name": "NN 3959", "stars": ["STAR_M"], "sector": [-3, 1, 2], "pos": [1.152, 1.312, 5.336]},
+    {"name": "Iota Pegasi", "stars": ["STAR_F"], "sector": [-3, -4, 2], "pos": [7.6, 1.472, 0.416]},
+    {"name": "Gliese 867", "stars": ["STAR_M", "STAR_M"], "sector": [-2, -4, -2], "pos": [6.84, 7.256, 6.072]},
+    {"name": "EZ Aquarii", "stars": ["STAR_M"], "sector": [-1, -2, -1], "pos": [4.272, 5.984, 5.072]},
+    {"name": "Ross 128", "stars": ["STAR_M"], "sector": [0, 1, 0], "pos": [0.584, 2.872, 0.152]},
+    {"name": "NN 3522", "stars": ["STAR_K"], "sector": [1, 1, 0], "pos": [2.232, 2.136, 2.144]},
+    {"name": "NN 3526", "stars": ["STAR_M"], "sector": [1, 1, 3], "pos": [0.656, 0.864, 6.776]},
+    {"name": "Sigma Draconis", "stars": ["STAR_K"], "sector": [-1, -1, 2], "pos": [1.984, 5.432, 1.64]},
+    {"name": "GJ 2034", "stars": ["WHITE_DWARF"], "sector": [1, -1, -3], "pos": [7.912, 0.656, 4.096]},
+    {"name": "GJ 2036", "stars": ["STAR_M", "STAR_M"], "sector": [2, -1, -4], "pos": [3.608, 2.152, 1.824]},
+    {"name": "Gliese 109", "stars": ["STAR_M"], "sector": [1, -3, 1], "pos": [6.6, 7.24, 2.616]},
+    {"name": "Alpha Fornacis", "stars": ["STAR_F", "STAR_G"], "sector": [3, -4, -3], "pos": [5.928, 5.072, 1.696]},
+    {"name": "Pollux", "stars": ["STAR_K_GIANT"], "sector": [3, 1, 1], "pos": [2.672, 5.2, 7.84]},
+    {"name": "Gliese 102", "stars": ["STAR_M"], "sector": [2, -3, 1], "pos": [1.944, 1.368, 5.432]},
+    {"name": "Gliese 103", "stars": ["STAR_K"], "sector": [2, -3, -4], "pos": [0.904, 2.816, 6.016]},
+    {"name": "Upsilon Andromedae", "stars": ["STAR_F"], "sector": [1, -4, 3], "pos": [5.504, 1.944, 5.056]},
+    {"name": "Luyten\"s Star", "stars": ["STAR_M"], "sector": [1, 0, 0], "pos": [3.448, 4.592, 1.128]},
```

Comment:
One final issue - the apostrophe-to-double quote path is converting this Lua-side escaped apostrophe into an escaped double-quote. Feel free to manually fix it (with a grep for other occurrences).